services/class-wc-ebanx-payment-validator.php

```diff
@@ -200,4 +201,15 @@ private function validate_payment_method_country() {
 
 		return false;
 	}
+
+	private function validate_instalments() {
+		$instalments = get_post_meta($this->order->id, '_ebanx_instalments', true);
+
+		if ( $instalments > 12 || $instalments < 1 ) {
+			$this->add_error(__('EBANX only supports instalments between 1 and 12'));
```

Comment:
Actually, let's make two different checks. One for lower than 1 and another for higher than 12. You can put both checking in the same method.